Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -118,57 +119,84 @@ export RGW_POOL_PREFIX=default
+
+2. Create [common-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/common-external.yaml) and [cluster-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/cluster-external.yaml)
+
+### Cluster Verification
+
+1. Verify the consumer cluster is connected to the source ceph cluster:
 
     ```console
     $ kubectl -n rook-ceph-external  get CephCluster
     NAME                 DATADIRHOSTPATH   MONCOUNT   AGE    STATE       HEALTH
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-6. StorageClass will also be created, verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+2.  Verify the creation of the storage class depending on the rbd pools and filesystem provided.
+    `ceph-rbd` and `cephfs` would be the respective names for the RBD and CephFS storage classes.
     ```console
     kubectl -n rook-ceph-external get sc
     ```
 
-7. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
-
-### CephCluster example (management)
+3. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
 
-The following CephCluster CR represents a cluster that will perform management tasks on the external cluster.
-It will not only act as a consumer but will also allow the deployment of other CRDs such as CephFilesystem or CephObjectStore.
-You would need to inject the admin keyring for that.
+### Connect to an External Object Store
 
-The corresponding YAML example:
+Create the object store resources:
+1. Create the [external object store CR](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml) to configure connection to external gateways.
+2. Create an [Object store user](https://github.com/rook/rook/blob/master/deploy/examples/object-user.yaml) for credentials to access the S3 endpoint.
+3. Create a [bucket storage class](https://github.com/rook/rook/blob/master/deploy/examples/storageclass-bucket-delete.yaml) where a client can request creating buckets.
+4. Create the [Object Bucket Claim](https://github.com/rook/rook/blob/master/deploy/examples/object-bucket-claim-delete.yaml), which will create an individual bucket for reading and writing objects.
 
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  dataDirHostPath: /var/lib/rook
-  cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6 # Should match external cluster version
+```console
+    $ cd deploy/examples
+    $ kubectl create -f object-external.yaml
+    $ kubectl create -f object-user.yaml
+    $ kubectl create -f storageclass-bucket-delete.yaml
+    $ kubectl create -f object-bucket-claim-delete.yaml
```

Comment:
```suggestion
    cd deploy/examples
    kubectl create -f object-external.yaml
    kubectl create -f object-user.yaml
    kubectl create -f storageclass-bucket-delete.yaml
    kubectl create -f object-bucket-claim-delete.yaml
```

when the user can directly copy and run the command no need to use `$`

Reply:
Okay, updated